vue/src/components/loki/LogLabelValuesCont.vue

```diff
@@ -52,23 +68,44 @@ export default defineComponent({
       }
     })
 
-    const internalLabels = computed((): Label[] => {
-      return labelValues.items.map((value: string): Label => {
-        return { name: value, selected: false }
+    const internalLabels = computed((): LabelItem[] => {
+      return labelValues.items.map((value: string): LabelItem => {
+        return { name: value, selected: false, label: props.label || '' }
```

Comment:
Looks like we could rename `LabelItem` to `Label` and adjust fields a bit to make them more readable:

```ts
export interface Label {
  name: string // label.name
  value: string // label.value
  selected: boolean
}
```

What do you think?

Reply:
unifyed as Label interface